README.md

```diff
@@ -50,3 +50,16 @@ visualize('MyComponent', () => {
     xsnap('ignore this test', <MyComponent/>);
 });
 ```
+
+## Testing asynchronious actions
+
+In order for the asynchronious tests to work, `eyes-storybook` must be configured in a way it knows to wait for the component to notify it is done. This is done by setting a flag on the relevant story and wait for applitools to look for this flag.
+
+Change your `applitools.config.js` file to the following:
+```js
+module.exports = {
+  waitBeforeScreenshots: `[data-test-ready="true"]`,
```

Comment:
So I actually created a `dataHooks` file for this.
I thought that users could import the datahook from the file, and use it
```
const { DATA_READY_HOOK } = require('storybook-snapper/dist/src/dataHooks');
module.exports = {
  waitBeforeScreenshots: `[${DATA_READY_HOOK}="true"]`,
  ...
}
```

Reply:
I don't like this, it's internal implementation and import is ugly. We should provide default configuration with some way to override it

Reply:
So what you're saying is that you want to have `eyes-storybook` as an internal implementation of this library?
And maybe add a binary that will run `eyes-storybook`?
And users will be able to add a config file with overrides?

So all configuration will be done in one place?

Hmmm 🤔 
I think I like the idea.
So no one will really need to deal with `eyes-storybook` themselves, and in the future, we might be able to change applitools for something else if we want.

wdyt>

Reply:
wrapping `eyes-storybook` sounds a bit too much at the moment. I actually thought exposing the storybook config instead like storybook's `webpack` config extension works.

```js
const {storybookSnapperApplitoolsConfig} = require('storybook-snapper');
module.exports = storybookSnapperApplitoolsConfig({
  //rest of the configuration
});
```

Reply:
I see.
I'm don't like having multiple "moving parts", but I guess as a start it's a good way to go

Reply:
I mean, what I initially wrote is also "moving parts", so I guess your approach will be better for now